sdks/js/3.x/docs/APIReference.md

```diff
@@ -49,6 +54,11 @@ ADB.Media.getInstance()
 var tracker = ADB.Media.getInstance();
 ```
 
+```javascript
+// create an instance with custom channel example
```

Comment:
Also add this line of comment
`// this overrides the channel which was set during the configuration`